cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -34,6 +36,23 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
           });
       }
     });
+
+    it('should render a placeholder image within a media block', () => {
+      cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+        ({ body }) => {
+          // `video` blocks can also contain audio, so this test
+          // checks if both media types have a placeholder image.
+          const media = getBlockData('video', body);
+          if (media) {
+            cy.get('div[class^="StyledVideoContainer"]').within(() => {
```

Comment:
See discussion here https://github.com/bbc/simorgh/pull/4198/files#r335390194 and on PR description.

Also see https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements - some other tests are using `data-e2e`

Reply:
Having discussed in person with @simonsinclair I'm happy to use these classes temporarily and create a separate issue to move them to use data attributes.

Reply:
I've created that issue and it lives here https://github.com/bbc/simorgh/issues/4297.